skills/declarative/calendarskill/dialogs/ShowMeetingDialog/ShowMeetingDialog.dialog

```diff
@@ -0,0 +1,369 @@
+              "property": "dialog.ShowMeeting_Subject",
+              "value": "=@Subject"
+            },
+            {
+              "$kind": "Microsoft.IfCondition",
+              "$designer": {
+                "id": "ezSgtj"
+              },
+              "condition": "dialog.ShowMeeting_FromDate == null",
+              "actions": [
+                {
+                  "$kind": "Microsoft.SetProperty",
+                  "$designer": {
+                    "id": "WJfTxT"
+                  },
+                  "property": "dialog.ShowMeeting_FromDate",
+                  "value": "=utcNow()"
+                }
+              ]
+            },
+            {
+              "$kind": "Microsoft.IfCondition",
+              "$designer": {
+                "id": "Plon6f"
+              },
+              "condition": "dialog.ShowMeeting_ToTime == null",
+              "actions": [
+                {
+                  "$kind": "Microsoft.SetProperty",
+                  "$designer": {
+                    "id": "Oa6oDZ"
+                  },
+                  "property": "dialog.ShowMeeting_FromTime",
+                  "value": "=dialog.ShowMeeting_FromDate"
+                }
+              ]
+            },
+            {
+              "$kind": "Microsoft.TraceActivity",
+              "$designer": {
+                "id": "U1v_VF"
+              }
+            },
+            {
+              "$kind": "Microsoft.SetProperty",
+              "$designer": {
+                "id": "R11s6N"
+              },
+              "property": "dialog.ShowMeeting_StartDateTime",
+              "value": "=formatDateTime((ticks(startOfDay(formatDateTime(dialog.ShowMeeting_FromDate)))+(ticks(formatDateTime(dialog.ShowMeeting_FromTime))-ticks(startOfDay(formatDateTime(dialog.ShowMeeting_FromTime))))-621355968000000000)/10000000)"
+            },
+            {
+              "$kind": "Microsoft.IfCondition",
+              "$designer": {
+                "id": "pkxlyy"
+              },
+              "condition": "dialog.ShowMeeting_ToDate == null",
+              "actions": [
+                {
+                  "$kind": "Microsoft.SetProperty",
+                  "$designer": {
+                    "id": "VMbx5N"
+                  },
+                  "property": "dialog.ShowMeeting_ToDate",
+                  "value": "=dialog.ShowMeeting_FromDate"
+                }
+              ]
+            },
+            {
+              "$kind": "Microsoft.IfCondition",
+              "$designer": {
+                "id": "CsFlKg"
+              },
+              "condition": "dialog.ShowMeeting_ToTime == null",
+              "actions": [
+                {
+                  "$kind": "Microsoft.SetProperty",
+                  "$designer": {
+                    "id": "Jlf29u"
+                  },
+                  "property": "dialog.ShowMeeting_ToTime",
+                  "value": "=formatDateTime('03/15/2018 23:59:59', 'yyyy-MM-ddTHH:mm:ss')"
```

Comment:
Why use this constant value, should we use a prompt to ask user for the time?

Reply:
If user didn't provide end time, we will use a end of day time stamp as default value.